sys/dev/efivar.h

```diff
@@ -32,6 +32,8 @@
 #include <machine/efi.h>
 
 struct efi_ops {
+	efi_status	(*efi_gettable)(struct uuid *, void **);
+	efi_status	(*efi_copytable)(struct uuid *, void **, size_t, size_t *);
```

Comment:
I don't see these two being assigned, otherwise looks fine.

Reply:
I will probably implement those in the near future, the whole EFI device driver is a WIP